### ISSUE TITLE
Add Docs Nest.js Fails

### ISSUE THREAD

**Issue by gasatrya** [continuedev/continue#2022]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: Windows 11 (WSL 2)
- Continue: v0.8.46
- IDE: VS Code 1.92.2
- Model:
- config.json:
```

### Description

Here's the docs url: https://docs.nestjs.com/

### Log output

```Shell
[Extension Host] Starting crawl from: https://docs.nestjs.com/ - Max Depth: 3
[Extension Host] Crawl completed
[Extension Host] Creating embeddings for 0 articles
ERR [Extension Host] No embeddings were created for site: https://docs.nestjs.com/
 Num chunks: 0
[Extension Host] No embeddings were created for site: https://docs.nestjs.com/
 Num chunks: 0
```

**Comment by sestinj** [comment]
Work is underway to solve this problem (the problem more generally being that you need a headless browser to parse sites relying on javascript, which NestJS docs do). We'll hopefully have a pre-release for VS Code in the next couple of days

**Comment by gasatrya** [comment]
Thanks @sestinj

**Comment by sestinj** [comment]
@tvajjhala Yup, we release a new indexer in main release this week, which should be able to successfully index almost any site. I have not tried Nest.js explicitly, but I don't know of any peculiarities that would cause it not to work
